data/json/items/chemicals_and_resources.json

```diff
@@ -1232,7 +1232,7 @@
     "fun": -20,
     "color": "white",
     "container": "bottle_glass",
-    "description": "Diethyl ether largely supplanted the use of chloroform as a general anesthetic due to ether's more favorable therapeutic index, that is, a greater difference between an effective dose and a potentially toxic dose.",
+    "description": "A useful industrial solvent, and the first inhaltive anesthetic in widespread use.  Its uncomfortable side effects and tendency to explode caused it to fall out of favor decades ago in the developed world.  If you can't get the good stuff, however, it might make a passable substitute.",
```

Comment:
```suggestion
    "description": "A useful industrial solvent, and the first inhalative anesthetic in widespread use.  Its uncomfortable side effects and tendency to explode caused it to fall out of favor decades ago in the developed world.  If you can't get the good stuff, however, it might make a passable substitute.",
```